catalogue/webapp/pages/item.js

```diff
@@ -385,13 +415,18 @@ ItemPage.getInitialProps = async (ctx: Context): Promise<Props> => {
     `https://wellcomelibrary.org/iiif/${sierraId}/manifest`
   )).json();
 
+  const canvases = manifest.sequences[0].canvases;
+  const currentCanvas = canvases[canvasIndex];
+  const canvasOcr = await getCanvasOcr(currentCanvas);
```

Comment:
I'd really like to setup some rules about how and when we decide to have 2 requests out for a single resource.
What's the reasoning here?

Reply:
To be clear why I don't like this is that if the service is slow, or takes seconds to time out, we've wed ourselves to that.

In this case, on top of another service which might have the same performance problems, thus serving a rather integral piece of the site slowly.

We saw that adding alt text asynchronously worked for screen readers, it would be interesting to see if it would work for SEO.

If we're going to stick with this, which feels like we're breaking the rule of 1 async request per resource, we could optimise it by not using JSON LD and rather URL construction: `https://wellcomelibrary.org/iiif/{sierraId}/contentAsText/{canvasId}`.